packages/hadron-build/lib/notary-service.js

```diff
@@ -0,0 +1,87 @@
+const download = require('download');
```

Comment:
to avoid confusion (since there's also a notary service) - could we rename the file `macos-notary-service`?

Reply:
the other notary-service will be removed as we are using garasign now. i've another upcoming PR to remove it completely and then we are left with this only.

Reply:
Renamed!